Description: Prevent SEGV after calling gs_image_class_1_simple.
 Prevent SEGV if memory allocation fails
 leading to penum being freed but then used again later.
 .
 Error created using :-
 MEMENTO_FAILAT=20913 ./membin/gpcl6 -sDEVICE=pbmraw -dMaxBitmap=2000
 -o /dev/null ./tests_private/pcl/pcl5cfts/fts.0100
Origin: upstream, http://git.ghostscript.com/?p=ghostpdl.git;h=0269a72
Author: Shailesh Mistry <shailesh.mistry@hotmail.co.uk>
Forwarded: yes
Bug: http://bugs.ghostscript.com/show_bug.cgi?id=697545
Last-Update: 2018-10-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/base/gxifast.c
+++ b/base/gxifast.c
@@ -110,9 +110,6 @@
                 penum->line = gs_alloc_bytes(penum->memory,
                                              penum->line_size, "image line");
                 if (penum->line == 0) {
-                    gx_default_end_image(penum->dev,
-                                         (gx_image_enum_common_t *) penum,
-                                         false);
                     return 0;
                 }
 #ifdef PACIFY_VALGRIND
